.vscode/settings.json

```diff
@@ -4,11 +4,15 @@
   },
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "editor.formatOnSave": true,
+  "editor.rulers": [100],
```

Comment:
I chose a value of 100 to match the existing TypeScript ecosystem. (This value needs to reflect the arbitrary value chosen for the project.)